core/src/SYCL/Kokkos_SYCL_WorkgroupReduction.hpp

```diff
@@ -21,9 +21,11 @@
 
 namespace Kokkos::Impl::SYCLReduction {
 
+// FIXME_SYCL It appears that using shuffles is slower than going through local
+// memory.
```

Comment:
Do you want to say more here?  Because if I understand correctly you are reported that it is slower for reductions on user-defined types (in your report that type is produced internally to handle the doing multiple reductions at once).  Because one might run an experiment on scalar types just like you did in your original work and conclude this is not needed any more.

Reply:
Sure, see c6dedc1c30a4af3ecfd56ef42bfe8286b2a8edd1.